src/extractors/withTranslationHOC.ts

```diff
@@ -168,6 +168,19 @@ function findTFunctionIdentifierInObjectPattern(
   return null;
 }
 
+/**
+ * Check whether a node path is the callee of a call expression.
+ *
+ * @param path the node to check.
+ * @returns true if the path is the callee of a call expression.
+ */
+function isCallee(path: BabelCore.NodePath): boolean {
```

Comment:
I think the return type could be a little more precise.

```suggestion
function isCallee(path: BabelCore.NodePath): path is BabelCore.NodePath<BabelTypes.CallExpression> {
```

Reply:
Done. Thanks.